simularium_metrics_calculator/tests/test_metrics_registry.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from simularium_metrics_calculator.metrics_registry import metrics_list
+
+
+def test_metrics_registry() -> None:
+    uids = set()
+    for metric in metrics_list:
+        uids.add(metric.uid)
+    unique_ids = list(uids)
+    assert len(unique_ids) == len(
+        metrics_list
+    ), "Multiple metrics have the same ID, make sure new metrics have a unique ID."
```

Comment:
If you got rid of the `uids` and just used the list index as the unique id when you call `create_metrics_registry` you wouldn't have to worry about people accidentally using the same ID when they make a new one

Reply:
yeah maybe I should do that, Dan also suggested these might only have to be dynamically generated

Reply:
now the uids are dynamically generated from the indices in the metrics list